torch/_inductor/cpp_builder.py

```diff
@@ -677,7 +664,7 @@ def _get_openmp_args(cpp_compiler):
             fb_openmp_extra_flags = f"-Wp,-fopenmp {openmp_lib}"
             passthough_args.append(fb_openmp_extra_flags)
 
-            libs.append("omp")
+            # libs.append("omp")
```

Comment:
Do we need to keep this?

Reply:
@jgong5 It seems no, according bin's log. `fb_openmp_extra_flags = f"-Wp,-fopenmp {openmp_lib}"` was setup `openmp` in `fb_code` env.